.github/workflows/charm-promotion.yaml

```diff
@@ -29,7 +34,7 @@ jobs:
         uses: canonical/charming-actions/release-charm@2.4.0
         with:
           credentials: ${{ secrets.CHARMCRAFT_CREDENTIALS }}
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          github-token: ${{ secrets.PAT_TOKEN }}
```

Comment:
we are using the wrong one, we should use promote-charm --> https://github.com/canonical/charming-actions/blob/main/promote-charm/action.yaml 


that doesn't need the github token

Reply:
I'll change it since it is more up to date, but the `release-charm` seems to be doing the same: https://github.com/canonical/charming-actions/blob/main/release-charm/README.md
We used it in the past (see https://github.com/canonical/oathkeeper-operator/blob/test-tls-config-file/.github/workflows/release.yaml, https://github.com/canonical/glauth-utils/commit/1fae6c7ae02f47e4d0a152283d2230cca280b212)